### PR TITLE
feat: update Order History changed to Orders & Subscriptions

### PR DESCRIPTION
**Pull request by mrazadar**
## Description

https://2u-internal.atlassian.net/browse/PON-88

This PR change the title Order History menu item title to Orders & Subscriptions.
- It ensures that the change is made for both Header and Learning Header

## Screenshots
Screenshot 2023-06-16 at 4 18 30 PM

### CODE REVIEW

File: src/Header.jsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const Header = ({ intl }) => {
   const orderHistoryItem = {
```
Comment (Syed-Ali-Abbas-Zaidi) [inline src/Header.jsx:81]: `orderHistoryItem` -> `orderSubscriptionItem` or `orderAndSubscriptionItem`

**Review by Syed-Ali-Abbas-Zaidi — APPROVED** [review]

**Comment by Syed-Ali-Abbas-Zaidi** [comment]
Hi @mrazadar! I just have a minor concern. Description says:
- `It ensures that this change only happens when SUBSCRIPTIONS_ORDERS_MENU_ITEM_ENABLED flag is enabled`

But I don't see `SUBSCRIPTIONS_ORDERS_MENU_ITEM_ENABLED` being used in the code. Can you please share your insights on this?

File: src/Header.jsx (hunk 1 above)
Comment (mrazadar) [reply src/Header.jsx]: updated.

**Comment by mrazadar** [comment]
> Hi @mrazadar! I just have a minor concern. Description says:
> 
> * `It ensures that this change only happens when SUBSCRIPTIONS_ORDERS_MENU_ITEM_ENABLED flag is enabled`
> 
> But I don't see `SUBSCRIPTIONS_ORDERS_MENU_ITEM_ENABLED` being used in the code. Can you please share your insights on this?

Previously, we wanted to guard this change against a flag but that is not necessary now. Since we're are very near to release date. 

Updated the PR description.

**Comment by Syed-Ali-Abbas-Zaidi** [comment]
LGTM!

**Pull request merged**